test/evergreen.yml

```diff
@@ -601,6 +610,11 @@ functions:
       params:
         file: ./wiredtiger/cmake_build/test/cppsuite/${test_name}.json
 
+  "upload wt2853_perf test stats":
```

Comment:
can we only have one `perf.send `function?

Reply:
Done! let me know how you think of it.